docs/general/administration/hardware-selection.md

```diff
@@ -97,6 +101,10 @@ In BIOS settings of motherboards, it may also be called `Smart Access Memory` or
 
 When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
 
+#### Software HDR to SDR Tone-mapping
+
+If there is no GPU available, the CPU can also be used to tone-map HDR content to SDR. **This is very demanding on hardware, therefore a GPU is ALWAYS recommended.** When using software tone-mapping, a software encoder will always be used. Tone-mapping 4K 60fps Dolby Vision content to SDR H264 requires a Ryzen 9 5950X for faster than real time transcoding. If you would like to output in H265 or AV1, you may end up in situations where no current desktop CPUs have enough performance.
```

Comment:
```suggestion
If there is no GPU available, the CPU can also be used to tone-map HDR content to SDR. **This can be very demanding on CPU, therefore a GPU is ALWAYS recommended.** When using software tone-mapping, a software encoder will always be used. Tone-mapping 4K 60fps Dolby Vision content to SDR H264 requires a Ryzen 9 5950X for faster than real time transcoding. If you would like to output in H265 or AV1, you may end up in situations where no current desktop CPUs have enough performance.
```

Reply:
Ideally this should be moved to a dedicated document about tone mapping but this is good enough for now.